cosmos.v1.proto

```diff
@@ -0,0 +1,474 @@
+syntax = "proto3";
+
+package cosmos.v1;
+
+option go_package = "github.com/figment-networks/tendermint-protobuf-def/cosmos";
+
+import "gogo.proto";
+import "google/protobuf/descriptor.proto";
+
+option(gogoproto.goproto_stringer_all) = false;
+option(gogoproto.stringer_all)         = false;
+
+enum SignMode {
+  SIGN_MODE_UNSPECIFIED = 0; SIGN_MODE_DIRECT = 1; SIGN_MODE_TEXTUAL = 2; SIGN_MODE_DIRECT_AUX = 3;
+  SIGN_MODE_LEGACY_AMINO_JSON                                                                  = 127;
```

Comment:
looks like the formatting is a bit off here

Reply:
You're right, I need to format this file thanks!